integration/tests/basic/rpc/test_rpc_base_calls.py

```diff
@@ -0,0 +1,650 @@
+        assert "Neon" in response["result"], "Invalid response result"
+
+    def test_net_version(self):
+        """Verify implemented rpc calls work work net_version"""
+        response = self.proxy_api.send_rpc("net_version")
+        assert "error" not in response
+        assert int(response["result"]) == self.web3_client._chain_id, \
+            f"Invalid response result {response['result']}"
+
+    @pytest.mark.parametrize("param", [Tag.LATEST, Tag.PENDING, Tag.EARLIEST, None])
+    def test_eth_get_transaction_count(self, param: tp.Union[Tag, None]):
+        """Verify implemented rpc calls work eth_getTransactionCount"""
+        if param:
+            self.send_neon(self.sender_account, self.recipient_account, 1)
+            param = [self.sender_account.address, param.value]
+        response = self.proxy_api.send_rpc("eth_getTransactionCount", params=param)
+        if not param:
+            assert "error" in response, "Error not in response"
+            return
+        assert "error" not in response
+        assert rpc_checks.is_hex(
+            response["result"]
+        ), AssertMessage.DOES_NOT_START_WITH_0X.value
+
+    def test_eth_send_raw_transaction(self):
+        """Verify implemented rpc calls work eth_sendRawTransaction"""
+        transaction = self.create_tx_object(amount=1)
+
+        signed_tx = self.web3_client.eth.account.sign_transaction(
+            transaction, self.sender_account.key
+        )
+        response = self.proxy_api.send_rpc(
+            "eth_sendRawTransaction", params=signed_tx.rawTransaction.hex()
+        )
+        assert "error" not in response
+        assert rpc_checks.is_hex(
+            response["result"]
+        ), f"Invalid response result {response['result']}"
+
+    def test_eth_sendRawTransaction_max_size(self):
+        """Validate max size for transaction, 127 KB"""
+        size = 127 * 1024
+        transaction = self.create_tx_object(amount=1)
+
+        transaction["data"] = gen_hash_of_block(size)
+        signed_tx = self.web3_client.eth.account.sign_transaction(
+            transaction, self.sender_account.key
+        )
+        response = self.proxy_api.send_rpc(
+            "eth_sendRawTransaction", params=signed_tx.rawTransaction.hex()
+        )
+        assert "error" not in response
+        assert rpc_checks.is_hex(
+            response["result"]
+        ), f"Invalid response result {response['result']}"
+
+    def test_eth_sendRawTransaction_max_contract_size(self):
+        """Validate max size for contract, 24 KB"""
+        contract, contract_deploy_tx = self.web3_client.deploy_and_get_contract(
+            "common/BigMemoryValue", "0.8.12",
+            contract_name="ValueOf24K", account=self.sender_account
+        )
+        assert rpc_checks.is_hex(contract.address)
+
+    @pytest.mark.parametrize("param", [32, 16, None])
+    def test_eth_get_transaction_by_hash_negative(self, param: tp.Union[int, None]):
```

Comment:
i think we could move eth_get_transaction_* all cases to test_rpc_get_transaction module